.circleci/images/primary/Dockerfile-jruby-9.3-latest

```diff
@@ -1,5 +1,6 @@
-# https://hub.docker.com/layers/jruby/library/jruby/9.3.4-jre11/images/sha256-1d34d94aa7f88994411f33075a57ab5d55384749977f932c3a8e470a6c5574e6?context=explore
-FROM jruby@sha256:1d34d94aa7f88994411f33075a57ab5d55384749977f932c3a8e470a6c5574e6
+# This image supports multiple platforms, see README for build instructions
+
+FROM jruby:9.3.4-jre11
```

Comment:
Minor: Maybe worth moving to JRuby 9.3.9.0 (the latest release)

Reply:
I propose we do each Ruby version updates in a dedicated PR, first to minimise impact, and also they would require updating the appraisal files.

Reply:
Done in https://github.com/DataDog/dd-trace-rb/pull/2412